app/assets/scss/application.scss

```diff
@@ -34,7 +34,6 @@ $path: "/static/images/";
 @import "toolkit/_grids.scss";
 @import "toolkit/_instruction-list.scss";
 @import "toolkit/_notification-banners.scss";
-@import "toolkit/_page-headings.scss";
```

Comment:
Unfortunately there are a lot of places where `page-heading` class is used directly, so we probably can't remove this line in this PR (unless you feel like also tackling those bits!).

Reply:
Not in the next 65 minutes :(

Reply:
:D no worries, another day then :D